docs/pages/build-reference/migrating.md

```diff
@@ -67,6 +67,8 @@ Only the second purpose applies with the new build system. All assets referenced
 
 ### Custom `"main"` entry point in **package.json** is not yet supported
 
+> Support for custom entry points is available in SDK +47
```

Comment:
```suggestion
> Support for custom entry points is available from SDK 47 and above versions.
```

Maybe, instead of a plus symbol, let's highlight the info in words? 😄